cmake/modules/FindPLUTO.cmake

```diff
@@ -8,10 +8,10 @@
 # - Try to find PLUTO instalation
 # Once done this will define
 #
```

Comment:
Geant3 should be exchanged by Pluto. This is an really old cut-n-paste error.

Reply:
done!